Update Image Reference Examples on p5js.org to ES6

Update the **Image** section of the [p5js reference](https://p5js.org/reference/) to ES6. This will entail updating all of the examples that fall under `@module Image`.

As a step forward in the transition to ES6 all of the reference examples on p5js.org should be updated to use ES6 standard. This will often mean simply switching 'var' to 'let' or in some cases [using es6 classes](https://github.com/processing/p5.js/issues/2215) where applicable. 

There are tons of resources online for learning about ES6. [Here is an interactive guide](http://stack.formidable.com/es6-interactive-guide/#/) that covers the biggest features.

Disregard previous question. I made the changes but I'm having problems passing the sample-linter (`./util/sample-linter.js`) when committing changes. I switched back to the master branch and hit the a similar error when running `npm run grunt` so I don't think it's my changes.

I'll dig into this problem later when I have more time but in the mean time if anyone has any pointers that would be helpful.

Error when trying to commit:
```
[10:47:44] node ./utils/sample-linter.js [started]
[10:47:45] node ./utils/sample-linter.js [failed]
[10:47:45] → × node ./utils/sample-linter.js found some errors. Please fix them and try committing again.

C:\Users\figraham\Code\github\p5.js\utils\sample-linter.js:174
            msg.column += this.lines[startLine].prefixLength;
                                                ^

TypeError: Cannot read property 'prefixLength' of undefined
    at Object.postprocess (C:\Users\figraham\Code\github\p5.js\utils\sample-linter.js:174:49)
    at linter.verifyAndFix.postprocess.problemLists (C:\Users\figraham\Code\github\p5.js\node_modules\eslint\lib\cli-engine.js:184:62)
    at Linter.verify (C:\Users\figraham\Code\github\p5.js\node_modules\eslint\lib\linter.js:1041:16)
    at Linter.verifyAndFix (C:\Users\figraham\Code\github\p5.js\node_modules\eslint\lib\linter.js:1135:29)
    at processText (C:\Users\figraham\Code\github\p5.js\node_modules\eslint\lib\cli-engine.js:178:32)
    at processFile (C:\Users\figraham\Code\github\p5.js\node_modules\eslint\lib\cli-engine.js:222:18)
    at fileList.map.fileInfo (C:\Users\figraham\Code\github\p5.js\node_modules\eslint\lib\cli-engine.js:537:20)
    at Array.map (<anonymous>)
    at CLIEngine.executeOnFiles (C:\Users\figraham\Code\github\p5.js\node_modules\eslint\lib\cli-engine.js:514:34)
    at eslintFiles (C:\Users\figraham\Code\github\p5.js\utils\sample-linter.js:219:23)
    at Object.<anonymous> (C:\Users\figraham\Code\github\p5.js\utils\sample-linter.js:236:16)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
```

Error from `npm run grunt` with no changes to master:
```
Running "eslint-samples:source" (eslint-samples) task
Warning: Cannot read property 'prefixLength' of undefined Use --force to continue.

Aborted due to warnings.
npm ERR! code ELIFECYCLE
npm ERR! errno 3
npm ERR! p5@0.7.2 grunt: `grunt`
npm ERR! Exit status 3
npm ERR!
npm ERR! Failed at the p5@0.7.2 grunt script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\figraham\AppData\Roaming\npm-cache\_logs\2018-12-13T18_05_50_650Z-debug.log
```
Note: checked log and it has no extra information.

--- a/src/image/image.js
+++ b/src/image/image.js
@@ -43,10 +43,10 @@ var p5 = require('../core/main');
  * @example
  * <div>
  * <code>
- * var img = createImage(66, 66);
+ * let img = createImage(66, 66);
  * img.loadPixels();
- * for (var i = 0; i < img.width; i++) {
- *   for (var j = 0; j < img.height; j++) {
+ * for (let i = 0; i < img.width; i++) {
+ *   for (let j = 0; j < img.height; j++) {
  *     img.set(i, j, color(0, 90, 102));
  *   }
  * }
@@ -57,10 +57,10 @@ var p5 = require('../core/main');
  *
  * <div>
  * <code>
- * var img = createImage(66, 66);
+ * let img = createImage(66, 66);
  * img.loadPixels();
- * for (var i = 0; i < img.width; i++) {
- *   for (var j = 0; j < img.height; j++) {
+ * for (let i = 0; i < img.width; i++) {
+ *   for (let j = 0; j < img.height; j++) {
  *     img.set(i, j, color(0, 90, 102, (i % img.width) * 2));
  *   }
  * }
@@ -72,12 +72,12 @@ var p5 = require('../core/main');
  *
  * <div>
  * <code>
- * var pink = color(255, 102, 204);
- * var img = createImage(66, 66);
+ * let pink = color(255, 102, 204);
+ * let img = createImage(66, 66);
  * img.loadPixels();
- * var d = pixelDensity();
- * var halfImage = 4 * (img.width * d) * (img.height / 2 * d);
- * for (var i = 0; i < halfImage; i += 4) {
+ * let d = pixelDensity();
+ * let halfImage = 4 * (img.width * d) * (img.height / 2 * d);
+ * for (let i = 0; i < halfImage; i += 4) {
  *   img.pixels[i] = red(pink);
  *   img.pixels[i + 1] = green(pink);
  *   img.pixels[i + 2] = blue(pink);
@@ -112,7 +112,7 @@ p5.prototype.createImage = function(width, height) {
  *  @example
  * <div class='norender notest'><code>
  * function setup() {
- *   var c = createCanvas(100, 100);
+ *   let c = createCanvas(100, 100);
  *   background(255, 0, 0);
  *   saveCanvas(c, 'myCanvas', 'jpg');
  * }
@@ -121,7 +121,7 @@ p5.prototype.createImage = function(width, height) {
  * // note that this example has the same result as above
  * // if no canvas is specified, defaults to main canvas
  * function setup() {
- *   var c = createCanvas(100, 100);
+ *   let c = createCanvas(100, 100);
  *   background(255, 0, 0);
  *   saveCanvas('myCanvas', 'jpg');
  *
@@ -225,7 +225,7 @@ p5.prototype.saveCanvas = function() {
  * }
  *
  * function mousePressed() {
- *   saveFrames('out', 'png', 1, 25, function(data) {
+ *   saveFrames('out', 'png', 1, 25, data => {
  *     print(data);
  *   });
  * }

--- a/src/image/loading_displaying.js
+++ b/src/image/loading_displaying.js
@@ -38,7 +38,7 @@ require('../core/error_helpers');
  * @example
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/laDefense.jpg');
  * }
@@ -51,7 +51,7 @@ require('../core/error_helpers');
  * <code>
  * function setup() {
  *   // here we use a callback to display the image after loading
- *   loadImage('assets/laDefense.jpg', function(img) {
+ *   loadImage('assets/laDefense.jpg', img => {
  *     image(img, 0, 0);
  *   });
  * }
@@ -148,7 +148,7 @@ function _sAssign(sVal, iVal) {
  * @example
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/laDefense.jpg');
  * }
@@ -161,7 +161,7 @@ function _sAssign(sVal, iVal) {
  * </div>
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/laDefense.jpg');
  * }
@@ -177,7 +177,7 @@ function _sAssign(sVal, iVal) {
  * <code>
  * function setup() {
  *   // Here, we use a callback to display the image after loading
- *   loadImage('assets/laDefense.jpg', function(img) {
+ *   loadImage('assets/laDefense.jpg', img => {
  *     image(img, 0, 0);
  *   });
  * }
@@ -185,7 +185,7 @@ function _sAssign(sVal, iVal) {
  * </div>
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/gradient.png');
  * }
@@ -319,7 +319,7 @@ p5.prototype.image = function(
  * @example
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/laDefense.jpg');
  * }
@@ -333,7 +333,7 @@ p5.prototype.image = function(
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/laDefense.jpg');
  * }
@@ -347,7 +347,7 @@ p5.prototype.image = function(
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/laDefense.jpg');
  * }
@@ -401,7 +401,7 @@ p5.prototype.tint = function() {
  * @example
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }
@@ -481,7 +481,7 @@ p5.prototype._getTintedImageCanvas = function(img) {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }
@@ -494,7 +494,7 @@ p5.prototype._getTintedImageCanvas = function(img) {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }
@@ -507,7 +507,7 @@ p5.prototype._getTintedImageCanvas = function(img) {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }

--- a/src/image/p5.Image.js
+++ b/src/image/p5.Image.js
@@ -38,28 +38,28 @@ var Filters = require('./filters');
  * @example
  * <div><code>
  * function setup() {
- *   var img = createImage(100, 100); // same as new p5.Image(100, 100);
+ *   let img = createImage(100, 100); // same as new p5.Image(100, 100);
  *   img.loadPixels();
  *   createCanvas(100, 100);
  *   background(0);
  *
  *   // helper for writing color to array
  *   function writeColor(image, x, y, red, green, blue, alpha) {
- *     var index = (x + y * width) * 4;
+ *     let index = (x + y * width) * 4;
  *     image.pixels[index] = red;
  *     image.pixels[index + 1] = green;
  *     image.pixels[index + 2] = blue;
  *     image.pixels[index + 3] = alpha;
  *   }
  *
- *   var x, y;
+ *   let x, y;
  *   // fill with random colors
  *   for (y = 0; y < img.height; y++) {
  *     for (x = 0; x < img.width; x++) {
- *       var red = random(255);
- *       var green = random(255);
- *       var blue = random(255);
- *       var alpha = 255;
+ *       let red = random(255);
+ *       let green = random(255);
+ *       let blue = random(255);
+ *       let alpha = 255;
  *       writeColor(img, x, y, red, green, blue, alpha);
  *     }
  *   }
@@ -93,7 +93,7 @@ p5.Image = function(width, height) {
    * @readOnly
    * @example
    * <div><code>
-   * var img;
+   * let img;
    * function preload() {
    *   img = loadImage('assets/rockies.jpg');
    * }
@@ -101,8 +101,8 @@ p5.Image = function(width, height) {
    * function setup() {
    *   createCanvas(100, 100);
    *   image(img, 0, 0);
-   *   for (var i = 0; i < img.width; i++) {
-   *     var c = img.get(i, img.height / 2);
+   *   for (let i = 0; i < img.width; i++) {
+   *     let c = img.get(i, img.height / 2);
    *     stroke(c);
    *     line(i, height / 2, i, height);
    *   }
@@ -120,7 +120,7 @@ p5.Image = function(width, height) {
    * @readOnly
    * @example
    * <div><code>
-   * var img;
+   * let img;
    * function preload() {
    *   img = loadImage('assets/rockies.jpg');
    * }
@@ -128,8 +128,8 @@ p5.Image = function(width, height) {
    * function setup() {
    *   createCanvas(100, 100);
    *   image(img, 0, 0);
-   *   for (var i = 0; i < img.height; i++) {
-   *     var c = img.get(img.width / 2, i);
+   *   for (let i = 0; i < img.height; i++) {
+   *     let c = img.get(img.width / 2, i);
    *     stroke(c);
    *     line(0, i, width / 2, i);
    *   }
@@ -164,15 +164,15 @@ p5.Image = function(width, height) {
    * values of the pixel at (1, 0). More generally, to set values for a pixel
    * at (x, y):
    * ```javascript
-   * var d = pixelDensity();
-   * for (var i = 0; i < d; i++) {
-   *   for (var j = 0; j < d; j++) {
+   * let d = pixelDensity();
+   * for (let i = 0; i < d; i++) {
+   *   for (let j = 0; j < d; j++) {
    *     // loop over
-   *     idx = 4 * ((y * d + j) * width * d + (x * d + i));
-   *     pixels[idx] = r;
-   *     pixels[idx+1] = g;
-   *     pixels[idx+2] = b;
-   *     pixels[idx+3] = a;
+   *     index = 4 * ((y * d + j) * width * d + (x * d + i));
+   *     pixels[index] = r;
+   *     pixels[index+1] = g;
+   *     pixels[index+2] = b;
+   *     pixels[index+3] = a;
    *   }
    * }
    * ```
@@ -184,10 +184,10 @@ p5.Image = function(width, height) {
    * @example
    * <div>
    * <code>
-   * var img = createImage(66, 66);
+   * let img = createImage(66, 66);
    * img.loadPixels();
-   * for (var i = 0; i < img.width; i++) {
-   *   for (var j = 0; j < img.height; j++) {
+   * for (let i = 0; i < img.width; i++) {
+   *   for (let j = 0; j < img.height; j++) {
    *     img.set(i, j, color(0, 90, 102));
    *   }
    * }
@@ -197,10 +197,10 @@ p5.Image = function(width, height) {
    * </div>
    * <div>
    * <code>
-   * var pink = color(255, 102, 204);
-   * var img = createImage(66, 66);
+   * let pink = color(255, 102, 204);
+   * let img = createImage(66, 66);
    * img.loadPixels();
-   * for (var i = 0; i < 4 * (width * height / 2); i += 4) {
+   * for (let i = 0; i < 4 * (width * height / 2); i += 4) {
    *   img.pixels[i] = red(pink);
    *   img.pixels[i + 1] = green(pink);
    *   img.pixels[i + 2] = blue(pink);
@@ -234,8 +234,8 @@ p5.Image.prototype._setProperty = function(prop, value) {
  * @method loadPixels
  * @example
  * <div><code>
- * var myImage;
- * var halfImage;
+ * let myImage;
+ * let halfImage;
  *
  * function preload() {
  *   myImage = loadImage('assets/rockies.jpg');
@@ -244,7 +244,7 @@ p5.Image.prototype._setProperty = function(prop, value) {
  * function setup() {
  *   myImage.loadPixels();
  *   halfImage = 4 * width * height / 2;
- *   for (var i = 0; i < halfImage; i++) {
+ *   for (let i = 0; i < halfImage; i++) {
  *     myImage.pixels[i + halfImage] = myImage.pixels[i];
  *   }
  *   myImage.updatePixels();
@@ -279,8 +279,8 @@ p5.Image.prototype.loadPixels = function() {
  *                              underlying canvas
  * @example
  * <div><code>
- * var myImage;
- * var halfImage;
+ * let myImage;
+ * let halfImage;
  *
  * function preload() {
  *   myImage = loadImage('assets/rockies.jpg');
@@ -289,7 +289,7 @@ p5.Image.prototype.loadPixels = function() {
  * function setup() {
  *   myImage.loadPixels();
  *   halfImage = 4 * width * height / 2;
- *   for (var i = 0; i < halfImage; i++) {
+ *   for (let i = 0; i < halfImage; i++) {
  *     myImage.pixels[i + halfImage] = myImage.pixels[i];
  *   }
  *   myImage.updatePixels();
@@ -331,8 +331,8 @@ p5.Image.prototype.updatePixels = function(x, y, w, h) {
  *                                    [R, G, B, A] or <a href="#/p5.Image">p5.Image</a>
  * @example
  * <div><code>
- * var myImage;
- * var c;
+ * let myImage;
+ * let c;
  *
  * function preload() {
  *   myImage = loadImage('assets/rockies.jpg');
@@ -373,10 +373,10 @@ p5.Image.prototype.get = function(x, y, w, h) {
  * @example
  * <div>
  * <code>
- * var img = createImage(66, 66);
+ * let img = createImage(66, 66);
  * img.loadPixels();
- * for (var i = 0; i < img.width; i++) {
- *   for (var j = 0; j < img.height; j++) {
+ * for (let i = 0; i < img.width; i++) {
+ *   for (let j = 0; j < img.height; j++) {
  *     img.set(i, j, color(0, 90, 102, (i % img.width) * 2));
  *   }
  * }
@@ -406,7 +406,7 @@ p5.Image.prototype.set = function(x, y, imgOrCol) {
  * @param {Number} height the resized image height
  * @example
  * <div><code>
- * var img;
+ * let img;
  *
  * function preload() {
  *   img = loadImage('assets/rockies.jpg');
@@ -499,10 +499,10 @@ p5.Image.prototype.resize = function(width, height) {
  * @param  {Integer} dh destination image height
  * @example
  * <div><code>
- * var photo;
- * var bricks;
- * var x;
- * var y;
+ * let photo;
+ * let bricks;
+ * let x;
+ * let y;
  *
  * function preload() {
  *   photo = loadImage('assets/rockies.jpg');
@@ -570,7 +570,7 @@ p5.Image.prototype.copy = function() {
  * @param {p5.Image} srcImage source image
  * @example
  * <div><code>
- * var photo, maskImage;
+ * let photo, maskImage;
  * function preload() {
  *   photo = loadImage('assets/rockies.jpg');
  *   maskImage = loadImage('assets/mask2.png');
@@ -636,8 +636,8 @@ p5.Image.prototype.mask = function(p5Image) {
  *                                to each filter, see above
  * @example
  * <div><code>
- * var photo1;
- * var photo2;
+ * let photo1;
+ * let photo2;
  *
  * function preload() {
  *   photo1 = loadImage('assets/rockies.jpg');
@@ -688,8 +688,8 @@ p5.Image.prototype.filter = function(operation, value) {
  * http://blogs.adobe.com/webplatform/2013/01/28/blending-features-in-canvas/
  * @example
  * <div><code>
- * var mountains;
- * var bricks;
+ * let mountains;
+ * let bricks;
  *
  * function preload() {
  *   mountains = loadImage('assets/rockies.jpg');
@@ -703,8 +703,8 @@ p5.Image.prototype.filter = function(operation, value) {
  * }
  * </code></div>
  * <div><code>
- * var mountains;
- * var bricks;
+ * let mountains;
+ * let bricks;
  *
  * function preload() {
  *   mountains = loadImage('assets/rockies.jpg');
@@ -718,8 +718,8 @@ p5.Image.prototype.filter = function(operation, value) {
  * }
  * </code></div>
  * <div><code>
- * var mountains;
- * var bricks;
+ * let mountains;
+ * let bricks;
  *
  * function preload() {
  *   mountains = loadImage('assets/rockies.jpg');
@@ -792,7 +792,7 @@ p5.Image.prototype.isModified = function() {
  * @param  {String} extension 'png' or 'jpg'
  * @example
  * <div><code>
- * var photo;
+ * let photo;
  *
  * function preload() {
  *   photo = loadImage('assets/rockies.jpg');

--- a/src/image/pixels.js
+++ b/src/image/pixels.js
@@ -29,15 +29,15 @@ require('../color/p5.Color');
  * contain the R, G, B, A values of the pixel at (1, 0). More generally, to
  * set values for a pixel at (x, y):
  * ```javascript
- * var d = pixelDensity();
- * for (var i = 0; i < d; i++) {
- *   for (var j = 0; j < d; j++) {
+ * let d = pixelDensity();
+ * for (let i = 0; i < d; i++) {
+ *   for (let j = 0; j < d; j++) {
  *     // loop over
- *     idx = 4 * ((y * d + j) * width * d + (x * d + i));
- *     pixels[idx] = r;
- *     pixels[idx+1] = g;
- *     pixels[idx+2] = b;
- *     pixels[idx+3] = a;
+ *     index = 4 * ((y * d + j) * width * d + (x * d + i));
+ *     pixels[index] = r;
+ *     pixels[index+1] = g;
+ *     pixels[index+2] = b;
+ *     pixels[index+3] = a;
  *   }
  * }
  * ```
@@ -60,11 +60,11 @@ require('../color/p5.Color');
  * @example
  * <div>
  * <code>
- * var pink = color(255, 102, 204);
+ * let pink = color(255, 102, 204);
  * loadPixels();
- * var d = pixelDensity();
- * var halfImage = 4 * (width * d) * (height / 2 * d);
- * for (var i = 0; i < halfImage; i += 4) {
+ * let d = pixelDensity();
+ * let halfImage = 4 * (width * d) * (height / 2 * d);
+ * for (let i = 0; i < halfImage; i += 4) {
  *   pixels[i] = red(pink);
  *   pixels[i + 1] = green(pink);
  *   pixels[i + 2] = blue(pink);
@@ -101,8 +101,8 @@ p5.prototype.pixels = [];
  *
  * @example
  * <div><code>
- * var img0;
- * var img1;
+ * let img0;
+ * let img1;
  *
  * function preload() {
  *   img0 = loadImage('assets/rockies.jpg');
@@ -116,8 +116,8 @@ p5.prototype.pixels = [];
  * }
  * </code></div>
  * <div><code>
- * var img0;
- * var img1;
+ * let img0;
+ * let img1;
  *
  * function preload() {
  *   img0 = loadImage('assets/rockies.jpg');
@@ -131,8 +131,8 @@ p5.prototype.pixels = [];
  * }
  * </code></div>
  * <div><code>
- * var img0;
- * var img1;
+ * let img0;
+ * let img1;
  *
  * function preload() {
  *   img0 = loadImage('assets/rockies.jpg');
@@ -195,7 +195,7 @@ p5.prototype.blend = function() {
  *
  * @example
  * <div><code>
- * var img;
+ * let img;
  *
  * function preload() {
  *   img = loadImage('assets/rockies.jpg');
@@ -289,7 +289,7 @@ p5.prototype.copy = function() {
  * @example
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }
@@ -302,7 +302,7 @@ p5.prototype.copy = function() {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }
@@ -315,7 +315,7 @@ p5.prototype.copy = function() {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }
@@ -328,7 +328,7 @@ p5.prototype.copy = function() {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }
@@ -341,7 +341,7 @@ p5.prototype.copy = function() {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }
@@ -354,7 +354,7 @@ p5.prototype.copy = function() {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }
@@ -367,7 +367,7 @@ p5.prototype.copy = function() {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }
@@ -380,7 +380,7 @@ p5.prototype.copy = function() {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/bricks.jpg');
  * }
@@ -427,17 +427,17 @@ p5.prototype.filter = function(operation, value) {
  * Getting the color of a single pixel with get(x, y) is easy, but not as fast
  * as grabbing the data directly from <a href="#/p5/pixels">pixels[]</a>. The equivalent statement to
  * get(x, y) using <a href="#/p5/pixels">pixels[]</a> with pixel density d is
- * <code>
- * var x, y, d; // set these to the coordinates
- * var off = (y * width + x) * d * 4;
- * var components = [
+ * ```javascript
+ * let x, y, d; // set these to the coordinates
+ * let off = (y * width + x) * d * 4;
+ * let components = [
  *   pixels[off],
  *   pixels[off + 1],
  *   pixels[off + 2],
  *   pixels[off + 3]
  * ];
  * print(components);
- * </code>
+ * ```
  * <br><br>
  * See the reference for <a href="#/p5/pixels">pixels[]</a> for more information.
  *
@@ -454,13 +454,13 @@ p5.prototype.filter = function(operation, value) {
  * @example
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/rockies.jpg');
  * }
  * function setup() {
  *   image(img, 0, 0);
- *   var c = get();
+ *   let c = get();
  *   image(c, width / 2, 0);
  * }
  * </code>
@@ -468,13 +468,13 @@ p5.prototype.filter = function(operation, value) {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/rockies.jpg');
  * }
  * function setup() {
  *   image(img, 0, 0);
- *   var c = get(50, 90);
+ *   let c = get(50, 90);
  *   fill(c);
  *   noStroke();
  *   rect(25, 25, 50, 50);
@@ -527,17 +527,17 @@ p5.prototype.get = function(x, y, w, h) {
  * @example
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/rockies.jpg');
  * }
  *
  * function setup() {
  *   image(img, 0, 0);
- *   var d = pixelDensity();
- *   var halfImage = 4 * (img.width * d) * (img.height * d / 2);
+ *   let d = pixelDensity();
+ *   let halfImage = 4 * (img.width * d) * (img.height * d / 2);
  *   loadPixels();
- *   for (var i = 0; i < halfImage; i++) {
+ *   for (let i = 0; i < halfImage; i++) {
  *     pixels[i + halfImage] = pixels[i];
  *   }
  *   updatePixels();
@@ -583,7 +583,7 @@ p5.prototype.loadPixels = function() {
  * @example
  * <div>
  * <code>
- * var black = color(0);
+ * let black = color(0);
  * set(30, 20, black);
  * set(85, 20, black);
  * set(85, 75, black);
@@ -594,9 +594,9 @@ p5.prototype.loadPixels = function() {
  *
  * <div>
  * <code>
- * for (var i = 30; i < width - 15; i++) {
- *   for (var j = 20; j < height - 25; j++) {
- *     var c = color(204 - j, 153 - i, 0);
+ * for (let i = 30; i < width - 15; i++) {
+ *   for (let j = 20; j < height - 25; j++) {
+ *     let c = color(204 - j, 153 - i, 0);
  *     set(i, j, c);
  *   }
  * }
@@ -606,7 +606,7 @@ p5.prototype.loadPixels = function() {
  *
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/rockies.jpg');
  * }
@@ -646,17 +646,17 @@ p5.prototype.set = function(x, y, imgOrCol) {
  * @example
  * <div>
  * <code>
- * var img;
+ * let img;
  * function preload() {
  *   img = loadImage('assets/rockies.jpg');
  * }
  *
  * function setup() {
  *   image(img, 0, 0);
- *   var d = pixelDensity();
- *   var halfImage = 4 * (img.width * d) * (img.height * d / 2);
+ *   let d = pixelDensity();
+ *   let halfImage = 4 * (img.width * d) * (img.height * d / 2);
  *   loadPixels();
- *   for (var i = 0; i < halfImage; i++) {
+ *   for (let i = 0; i < halfImage; i++) {
  *     pixels[i + halfImage] = pixels[i];
  *   }
  *   updatePixels();

